src/components/MovieCard.js

```diff
@@ -1,20 +1,45 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import { filterId, removeMovie } from "../store/actionCreators";
 
-const MovieCard = ({ image, title, year, description }) => {
+const MovieCard = (prop) => {
+  const dispatch = useDispatch();
   return (
-    <>
-      <div className="card">
-        <img className="card-image" src={image} alt={title} />
-        <div className="card-details">
-          <div>
-            <h4>{title}</h4>
-            <p>{description}</p>
-          </div>
-          <button>{year}</button>
+    <div
+      className="card"
+      onClick={() =>
+        dispatch(
```

Comment:
change to going to the /movies?id={...}, no need to do anything with store

Reply:
resolved